playground/src/main.tsx

```diff
@@ -248,12 +248,6 @@ async function loaded() {
 function registerMonacoLanguageServiceProviders(
   languageService: ILanguageService,
 ) {
-  monaco.languages.setLanguageConfiguration("qsharp", {
```

Comment:
I wasn't expecting this to get dropped. Are we not using this anymore?

Reply:
Nope! Good riddance. Called out in the PR description:

> removed wonky word pattern in playground and VS Code because we don't need it anymore now that we have proper `.` completions! Also update trigger characters.